### DISCUSSION TITLE
请教:layui table 导出时,隐藏标题栏的css 代码

### DISCUSSION THREAD

**Discussion by betterp2p** [layui/layui#1549] [问答] [answered]
![屏幕截图 2024-01-10 084021](https://github.com/layui/layui/assets/154388372/a81ec1e7-eaea-4fbc-b03b-4bf865b04a4e)
![屏幕截图 2024-01-10 084005](https://github.com/layui/layui/assets/154388372/58a65efc-ab07-4cdb-8111-f97654c68015)

请教大佬一下，怎么导出excel时屏蔽标题栏中的 css 代码，谢谢。

**Comment by Sight-wcg** [comment]
参考 col.exportTemplet
https://layui.dev/docs/2/table/#cols.templet

**Reply by betterp2p** [reply]
谢谢您的回复。惭愧，查看了文档还是没有解决。我是想把title里面的css 在导出时去掉
`      {field:'aa',  title: '<span class=layui-font-green title=一次评价>一次Ｏ</span>',templet:'#aaTpl'},
      {field:'abj',  title: '<span class=layui-font-green title=一次评价>一次Ｅ </span>',templet:'#abjTpl'},
`

**Reply by Sight-wcg** [reply, marked as answer]
不好意思，这个场景应该用 col.fieldTitle 而不是 col.exportTemplet

fieldTitle 设置列的字段标题。该属性在筛选列和导出场景中优先级高于 title 属性。

```
title:'<h3>用户名<h3>',
fieldTitle:'用户名'
```

**Reply by betterp2p** [reply]
没注意到版本的升级增加了些属性。谢谢您。
